docs/how-to/kubernetes.md

```diff
@@ -162,26 +178,50 @@ The client sends an image to the exposed `Flow` on `$EXTERNAL_IP` and retrieves
 Finally, it prints the uri of the closest matches.
 
 ```python
-import requests
-from jina import DocumentArray
 import os
+
+from jina.clients import Client
+from jina import DocumentArray
+
 host = os.environ['EXTERNAL_IP']
 port = 80
-url = f'http://{host}:{port}'
 
-doc = DocumentArray.from_files('./imgs/*.jpg').apply(lambda d: d.load_uri_to_image_blob())[0].to_dict()
-resp = requests.post(f'{url}/search', json={'data': [doc]})
-matches = resp.json()['data']['docs'][0]['matches']
-print(f'Matched documents: {len(matches)}')
+client = Client(host=host, port=port)
+client.show_progress = True
+docs = DocumentArray.from_files("./imgs/*.png").apply(
+    lambda d: d.load_uri_to_image_tensor()
+)
+resp_query = client.post("/search", inputs=docs, return_results=True)
+
+matches = resp_query[0].docs[0].matches
```

Comment:
Is this correct for 3.0?

Reply:
yes, when the client is used stand alone